inference-engine/src/mkldnn_plugin/mkldnn_memory.h

```diff
@@ -34,20 +39,15 @@ namespace MKLDNNPlugin {
  * Represent internal plugin abstraction of tensor description
  *
  */
-class MKLDNNMemoryDesc {
+class MKLDNNMemoryDesc : public MemoryDesc {
```

Comment:
We need to clean up MKLDNNMemoryDesc API

Reply:
As it was discussed, the API cleanup is planned in a separate PR.